src/dndbeyond/content-scripts/character.js

```diff
@@ -527,6 +536,10 @@ function rollAction(paneClass) {
                     damage_types.push("Sneak Attack");
                 }
             }
+            if (character.hasClassFeature("Song of Victory") && character.getSetting("wizard-bladesong", false)) {
+                damages.push(String(character.getAbility("INT").mod));
```

Comment:
I would verify that the value is available (I don't know, in case it never parsed the abilities?) and isn't 0. There's no need to have a constant damage of 0 if the character has no intelligence

Reply:
Fixed